src/compactor.rs

```diff
@@ -56,7 +57,7 @@ impl Compactor {
                 }
             };
             err_tx.send(Ok(())).expect("err channel failure");
-            orchestrator.run();
+            orchestrator.run(c);
```

Comment:
WDYT about passing compression codec into CompactorOrchestrator::new, so we don't have to modify all of its functions (run, load, and so on)?

Reply:
Actually, I don't think you'd need any of these code change if you used SsTableFormat since TableStore has SsTableFormat in it already.